test/checks/shared/filename-is-valid-accessible-name.js

```diff
@@ -0,0 +1,45 @@
+describe('filename-is-valid-accessible-name tests', function() {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var queryFixture = axe.testUtils.queryFixture;
+	var checkContext = axe.testUtils.MockCheckContext();
+	var check = checks['filename-is-valid-accessible-name'];
+
+	afterEach(function() {
+		fixture.innerHTML = '';
+		checkContext.reset();
+	});
+
+	it('returns undefined when element accessible name contains image extension type', function() {
+		var vNode = queryFixture(
+			'<img id="target" src="https://www.w3.org/WAI/demos/bad/img/w3c" alt="w3c.png" />'
+		);
+		var actual = check.evaluate.call(checkContext, vNode.actualNode);
+		assert.isUndefined(actual);
+	});
+
+	it('returns undefined when element accessible name contains image extension type', function() {
```

Comment:
This is the same text as the prior test. Should update it to reflect that you're testing a different element (`<input type="image">`)